src/Operation/Watch.php

```diff
@@ -264,7 +264,7 @@ public function __construct(Manager $manager, $databaseName, $collectionName, ar
 
         $this->manager = $manager;
         $this->databaseName = (string) $databaseName;
-        $this->collectionName = isset($collectionName) ? (string) $collectionName : null;
+        $this->collectionName = $collectionName ?? null;
```

Comment:
`?? null` can be removed here. The previous `(string)` cast and ternary was only used to avoid turning a null value into an empty string.

Reply:
Good point - removed this.